tuna/example/session.py

```diff
@@ -25,25 +25,30 @@
 #
 ###############################################################################
 """Session table and its associate functionality"""
+
+import argparse
+from typing import Tuple
 from sqlalchemy import UniqueConstraint
 
+from sqlalchemy.orm.query import Query
 from tuna.dbBase.base_class import BASE
 from tuna.session_mixin import SessionMixin
+from tuna.worker_interface import WorkerInterface
 
 
 class SessionExample(BASE, SessionMixin):
   """Session table to keep track of tunning sesions"""
   #pylint: disable=attribute-defined-outside-init
 
-  __tablename__ = "session_example"
-  __table_args__ = (UniqueConstraint("arch",
-                                     "num_cu",
-                                     "rocm_v",
-                                     "reason",
-                                     "docker",
-                                     name="uq_idx"),)
+  __tablename__: str = "session_example"
+  __table_args__: Tuple[UniqueConstraint] = (UniqueConstraint("arch",
+                                                              "num_cu",
+                                                              "rocm_v",
+                                                              "reason",
+                                                              "docker",
+                                                              name="uq_idx"),)
 
-  def get_query(self, sess, sess_obj, entry):
+  def get_query(self, sess: SessionMixin, sess_obj, entry) -> Query:
```

Comment:
missing typehints for sess_obj and entry

Reply:
The session name aliasing is biting here.
sess is a sqlalchemy session
sess_obj is a SessionMixin
I think entry is an arg list